### PR TITLE
Fix/more icml issues

### PR DESCRIPTION
**Pull request by melisabok**
- Fixes https://github.com/openreview/openreview-py/issues/1550
- Fixes https://github.com/openreview/openreview-py/issues/1546
- Fix Bid invitation so PCs can edit bid edges
- Fix AC console variable names

### CODE REVIEW

File: openreview/venue/webfield/areachairsWebfield.js [hunk 1]
```diff
@@ -17,18 +17,18 @@ return {
...
-      edgeBrowserDeployedUrl: `/edges/browse?start=${areaChairsId}/-/Assignment,tail:${user.id}&traverse=${reviewerGroup}/-/Assignment&edit=${reviewerGroup}/-/Invite_Assignment&browse=${reviewerGroup}/-/Affinity_Score;${reviewerGroup}/-/Bid;${reviewerGroup}/-/Custom_Max_Papers,head:ignore;${reviewerGroup}/-/Reviews_Submitted,head:ignore&hide=${reviewerGroup}/-/Conflict&maxColumns=2&referrer=[AC%20Console](/group?id=${areaChairsId})`,
+      edgeBrowserProposedUrl: `/edges/browse?start=${domain.content.area_chairs_assignment_id.value},tail:${user.id}&traverse=${domain.content.reviewers_proposed_assignment_id.value},label:${reviewerAssignmentTitle}&edit=${domain.content.reviewers_proposed_assignment_id.value},label:${reviewerAssignmentTitle};${domain.content.reviewers_invite_assignment_id.value}&browse=${reviewerGroup}/-/Aggregate_Score,label:${reviewerAssignmentTitle};${domain.content.reviewers_affinity_score_id.value};${domain.content.bid_name.value};${domain.content.reviewers_custom_max_papers_id.value},head:ignore&hide=${domain.content.reviewers_conflict_id.value}&maxColumns=2&referrer=[AC%20Console](/group?id=${areaChairsId})`,
```
Comment (celestemartinez) [inline openreview/venue/webfield/areachairsWebfield.js:20]: what is this invitation we removed: `${reviewerGroup}/-/Reviews_Submitted`?

File: openreview/venue/webfield/areachairsWebfield.js [hunk 2]
```diff
@@ -17,18 +17,18 @@ return {
...
+      edgeBrowserDeployedUrl: `/edges/browse?start=${domain.content.area_chairs_assignment_id.value},tail:${user.id}&traverse=${domain.content.reviewers_assignment_id.value}&edit=${domain.content.reviewers_invite_assignment_id.value}&browse=${domain.content.reviewers_affinity_score_id.value};${reviewerGroup}/-/${domain.content.bid_name.value};${domain.content.reviewers_custom_max_papers_id.value},head:ignore&hide=${domain.content.reviewers_conflict_id.value}&maxColumns=2&referrer=[AC%20Console](/group?id=${areaChairsId})`,
     },
     submissionInvitationId: domain.content.submission_id.value,
-    seniorAreaChairsId: '',
-    areaChairName: 'Area_Chairs',
+    seniorAreaChairsId: domain.content.senior_area_chairs_id.value,
```
Comment (celestemartinez) [inline openreview/venue/webfield/areachairsWebfield.js:24]: would this give an error if there are no SACs in the venue?

File: openreview/venue/webfield/areachairsWebfield.js (hunk 1 above)
Comment (melisabok) [reply openreview/venue/webfield/areachairsWebfield.js]: Reviews Submitted is only being used by TMLR. I don't think the venue class supports an edge that counts the reviews submitted

**Review by celestemartinez — APPROVED** [review]

**Comment by melisabok** [comment]
I'm going to make more changes in the AC console, bid and reviewer recommendations are optional edges so we should add them to the browse url if they exist

**Pull request merged**